src/main/java/org/openstreetmap/atlas/checks/validation/intersections/LineCrossingWaterBodyCheck.java

```diff
@@ -412,12 +425,18 @@ private boolean canCrossWaterBody(final AtlasItem crossingItem,
                 final Set<Tuple<PolyLine, Set<Location>>> interactionsPerWaterbodyComponent = this
                         .getInteractionsPerWaterbodyComponent(waterbody, object,
                                 lineItem.asPolyLine(), flag);
-                final AtomicBoolean validCrossWaterbody = new AtomicBoolean(true);
+                this.validCrossWaterbody.getAndSet(true);
```

Comment:
What was the reason for this change?

Reply:
I moved validCrossWaterbody to global variable because I need an access from handlingFootPath method.

Reply:
I see, ok.